src/shims/dir.props

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\dir.props" />
+    <PropertyGroup>
+    <BuildConfigurations>
+      netcoreapp;
+      uap;
+      uapaot;
+    </BuildConfigurations>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <NetFxReference Include="mscorlib" />
+    <NetFxReference Include="System" />
+    <NetFxReference Include="System.Core" />
+    <NetFxReference Include="System.ComponentModel.Composition" />
+    <NetFxReference Include="System.Data" />
+    <NetFxReference Include="System.Drawing" />
+    <NetFxReference Include="System.IO.Compression.FileSystem" />
+    <NetFxReference Include="System.Numerics" />
+    <NetFxReference Include="System.Runtime.Serialization" />
+    <NetFxReference Include="System.Web" />
+    <NetFxReference Include="System.Xml" />
+    <NetFxReference Include="System.Xml.Linq" />
+  </ItemGroup>
+
+  <PropertyGroup>
+  	<GenFacadesOutputPath>$(IntermediateOutputPath)facades/</GenFacadesOutputPath>
```

Comment:
Sharing this in this manner won't work, as IntermediateOutputPath will be different when run in the context of shims.proj and apicompat.proj.